### DISCUSSION TITLE
Verifying Student Developer Pack Using Persian Based Date.

### DISCUSSION THREAD

**Discussion by mazdakdev** [community/community#131965] [GitHub Education]
### Hey !

I was wondering, is it possible to verify using a school student card that is entirely in a non-English language and follows the Persian calendar date format?

**Comment by Aditijainnn** [comment]
1- Set GitHub profile.

    2- Username: Nickname
       Name : Real name in English
      bio:Name of University in English(This name is same as Github Education application form)

    3- Scan  Student ID(Written in any language)

    4- Crop Student ID in the scan image.

    5- Edit Student ID's scan image to write English translation.

    6- Write the translation in red text next to the original language text.

    7- informations should be translated.
    Name of University(in English)
    Real Name(in English)
    Valid Date (in English)(example: "valid until 04/2024")

   8-  Display the translated Student ID in the PC Monitor.
        ~ Open "The GitHub Student Developer Pack"'s upload page using Smartphone.
        ~ Take photo of the PC Monitor(it's displayed Student ID) using Smartphone.

  9-  Submit.

**Comment by samus-aran** [comment]
Hi @mazdakdev 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
